node/rest/src/routes.rs

```diff
@@ -334,6 +335,13 @@ impl<N: Network, C: ConsensusStorage<N>, R: Routing<N>> Rest<N, C, R> {
             return Err(RestError(format!("Unable to broadcast transaction '{}' (node is syncing)", fmt_id(tx.id()))));
         }
 
+        // If the transaction exceeds the transaction size limit, return an error.
+        // TODO: Should this be a blocking task?
```

Comment:
Could we spawn a blocking task here?

Reply:
I think `write_le` should be fast even for large txs. The only way to be sure is to hit the endpoint with a very large tx and measure the call, not sure if we could do that as a burn-in? Otherwise I suggest we leave this for a follow-up.

Reply:
Basically I think a benchmark should be run on `LimitedWriter::new`. That should quickly tell us if running too many of these in the tokio executor would block it.

Reply:
@ljedrz @pauan 

Could you comment on what the better call is?

Reply:
I'm not sure what the cost is here at first glance, but the rule of thumb would be that anything that can go into the millisecond scale should be done within a blocking task; several microseconds can be ignored.

`tokio-console` is also quite good at detecting which tasks spend too much time blocking, in case we ever miss/suspect something.

Reply:
I'd avoid using the blocking thread pool for anything that isn't truly expensive, a couple ms isn't super worrisome imo especially as this is a larger tx than our current limit of 128kb?

Reply:
In any case I think we should confirm by testing on devnet (without making this a blocker for merging). It'll go well with our current testing of the REST server.

Reply:
15 ms is a *lot*... in general it's a bad idea for async code to block for that long, because it blocks the executor and can prevent *other* async tasks from running concurrently.